templates/hotgraphicPopup.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="hotgraphic-popup-toolbar component-item-color clearfix">
+    {{#unless _hidePagination}}
+    <div class="hotgraphic-popup-nav component-item-color">
+        <button class="base hotgraphic-popup-controls back" aria-label="{{_globals._accessibility._ariaLabels.previous}}" role="button">
+            <div class="hotgraphic-popup-arrow-l component-item-text-color icon icon-controls-small-left"></div>
+        </button>
+        <div class="hotgraphic-popup-count component-item-text-color">
+            <span class="current">1</span>/<span class="total">{{_items.length}}</span>
+        </div>
+        <button class="base hotgraphic-popup-controls next" aria-label="{{_globals._accessibility._ariaLabels.next}}" role="button">
+            <div class="hotgraphic-popup-arrow-r component-item-text-color icon icon-controls-small-right"></div>
+        </button>
+    </div>
+    {{/unless}}
+    <button class="base hotgraphic-popup-done" aria-label="{{_globals._accessibility._ariaLabels.closePopup}}" role="button">
+        <div class="hotgraphic-popup-close component-item-text-color icon icon-cross"></div>
+    </button>
+</div>
+<div class="hotgraphic-popup-inner clearfix">
+    {{#each _items}}
+    <div class="hotgraphic-item component-item item-{{@index}} {{_classes}}">
+        <div class="hotgraphic-item-graphic">
+            <div class="hotgraphic-item-graphic-inner">
+            <img src="{{_graphic.src}}" aria-label="{{_graphic.alt}}" tabindex="0"/>
+            </div>
+            {{#if _graphic.attribution}}
+            <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
+            {{/if}}
+        </div>
+        <div class="hotgraphic-item-content">
+            <div class="hotgraphic-item-content-inner">
+            <div class="hotgraphic-content-title" role="heading" aria-level="5" tabindex="0">
+                {{{title}}}
+            </div>
+            <div class="hotgraphic-content-body">
+                {{{a11y_text body}}}
+            </div>
+            </div>
+        </div>
+    </div>
+    {{/each}}
+</div>
```

Comment:
can you add `{{a11y_wrap_focus}}` to the last line?

Reply:
done